specification/metrics/purefa-metrics.md

```diff
@@ -167,6 +165,16 @@ This document describes the semantic conventions for Pure FlashArray Metrics.
+| Available | purefa_network_interface_speed_bandwidth_bytes          | FlashArray network interface speed in bytes per second      | bytes/second  | Gauge                                                                                                                   | Double     | `name`        | (interface name)                                                                                                                                                                                                                                                                                                                      |
+|           |                                                      |                                                               |               |                                                                                                                         |            | `enabled`        | `true`, `false`                                                                                                                                                                                                                                                                                                                           |
+|           |                                                      |                                                               |               |                                                                                                                         |            | `ethsubtype`        | `bond`, `physical`, `vif`                                                                                                                                                                                                                                                                                                                           |
+|           |                                                      |                                                               |               |                                                                                                                         |            | `services`        | (service name)                                                                                                                                                                                                                                                                                                                           |
```

Comment:
I think this and `type` are in the wrong section. Should be in the port_info below not in performance

Reply:
It was decided to be as compliant with the Purity API as much as possible.